systemd/ceph-volume@.service

```diff
@@ -7,7 +7,7 @@ Wants=local-fs.target
 Type=oneshot
 KillMode=none
 Environment=CEPH_VOLUME_TIMEOUT=10000
-ExecStart=/bin/sh -c 'timeout $CEPH_VOLUME_TIMEOUT /usr/sbin/ceph-volume-systemd %i'
+ExecStart=/bin/sh -c 'timeout $CEPH_VOLUME_TIMEOUT /usr/bin/ceph-volume-systemd %i'
```

Comment:
both `ceph.spec.in` and `debian/ceph-osd.install` install `ceph-volume-systemd` into `/usr/sbin/`. so this change would break the functionality of the rpm and deb packages.

could you try out following patch instead?
```diff
diff --git a/cmake/modules/Distutils.cmake b/cmake/modules/Distutils.cmake
index 8092e58379c..ff01aaf818a 100644
--- a/cmake/modules/Distutils.cmake
+++ b/cmake/modules/Distutils.cmake
@@ -25,9 +25,9 @@ function(distutils_install_module name)
       list(APPEND options
         --root=\$ENV{DESTDIR}
         --single-version-externally-managed)
-      if(NOT \"${DU_INSTALL_SCRIPT}\" STREQUAL \"\")
-        list(APPEND options --install-script=${DU_INSTALL_SCRIPT})
-      endif()
+    endif()
+    if(NOT \"${DU_INSTALL_SCRIPT}\" STREQUAL \"\")
+      list(APPEND options --install-script=${DU_INSTALL_SCRIPT})
     endif()
     execute_process(
     COMMAND ${Python3_EXECUTABLE}
```

Reply:
Will try and will let you know

Reply:
Your patch did the trick. ceph-volume-systemd is now installed into /usr/sbin and systemd unit file points to correct location. So in my opinion we should adopt your patch to fix this issue. Any suggestions for commit message?